quicksort/Cpp/quick_sort.cpp

```diff
@@ -0,0 +1,42 @@
+
+#include <iostream>
+
+using std::cout;
+
+void quick_sort(int a[], int left, int right) {
+
```

Comment:
Redundant blank line at the start of a code block should be deleted.  [whitespace/blank_line] [2]

*Origin: CPPLintBear, Section: `all.cpplint`.*